src/HotChocolate/CodeGeneration/src/CodeGeneration.Neo4J/HotChocolate.CodeGeneration.Neo4J.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <IsPackable>false</IsPackable>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
Are these required for our "CodeGeneration" libraries? I feel like this might be something that can be dropped and was just for the analyzer?

Reply:
I think this might also be in the HotChocolate.CodeGeneration.csproj, so if we remove shud do it there too.

Reply:
This is only needed in the analyzer